### PR TITLE
[DEN-258] Create external cli project using cobra and viper

### PR DESCRIPTION
**Pull request by ssaluja4**

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/*
+Copyright © 2022 NAME HERE <EMAIL ADDRESS>
```
Comment (stevenpaz) [inline cmd/root.go]: We should fill out this placeholder copyright line

File: cmd/root.go [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+func Execute() {
+	rootCmd := &cobra.Command{
+		Use:   "EC CLI",
+		Short: "EC CLI",
+		Long: `A CLI library for Go that empowers users to onboard edgecast
+		products and services.`,
+	}
+
+	if err := rootCmd.Execute(); err != nil {
+		os.Exit(1)
```
Comment (defstream) [inline cmd/root.go]: The error should be logged log.Fatal(err), which will print and exit with an error code of 1.

File: main.go [hunk 3]
```diff
@@ -0,0 +1,12 @@
+// Copyright 2021 Edgecast Inc., Licensed under the terms of the Apache 2.0 license.
```
Comment (defstream) [inline main.go]: Update to 2022

**Review by defstream — CHANGES_REQUESTED** [review]

File: cmd/root.go (hunk 2 above)
Comment (ssaluja4) [reply cmd/root.go]: Updated. thanks.

File: main.go (hunk 3 above)
Comment (ssaluja4) [reply main.go]: Updated. Thanks.

**Review by defstream — APPROVED** [review]

**Pull request merged**